server/db-patches/patch-23-22.sql

```diff
@@ -0,0 +1,5 @@
+-- It is not possible to remove a value from an enum type with ALTER.
+-- Moreover, any records that's using the new enum value must be updated
+-- or deleted.  So we are not reverting the shot_blocK_type change here.
```

Comment:
It's a little bit involved, but you can remove enum values: https://blog.yo1.dog/updating-enum-values-in-postgresql-the-safe-and-easy-way/

Reply:
Yes, I'm aware we could replace the old enum with a new one.  But as stated in the comment, we would still need to delete the shots that's been marked with the watchdog block type (or change to some other value that exist in the new enum) since the new enum wouldn't have that value.

I suppose we try it and let it error out if any shots still using the watchdog block type?